src/yunohost/tools.py

```diff
@@ -165,59 +165,10 @@ def tools_adminpw(new_password, check_strength=True):
 
 
 @is_unit_operation()
```

Comment:
I think we don't need this decorator here anymore since it got moved to the new function

Reply:
Are you sure? The other command will take care of that?

Reply:
Hmyep since it has the decorator ?

Reply:
I guess so, I've removed it then.